src/hooks/fileTreeHooks/useCreate.tsx

```diff
@@ -0,0 +1,21 @@
+import {useAppDispatch} from '@redux/hooks';
+import {openCreateFolderModal, openNewResourceWizard} from '@redux/reducers/ui';
+
+export const useCreate = () => {
+  const dispatch = useAppDispatch();
+
+  const onCreateFolder = (absolutePath: string) => {
```

Comment:
Currently this doesn't exist on main anymore, was changed to `onCreateFileFolder`. We'll need to do some refactoring as we will have conflicts on merging

Reply:
rebased and fixed